src/main/kotlin/step2/Calculator.kt

```diff
@@ -0,0 +1,27 @@
+package step2
+
+class Calculator {
+    private var total = 0
+
+    fun calculate(expression: String): Int {
+        val elements = expression.split(" ").groupBy { isNumeric(it) }
+        val numbers = elements[true]?.map { it.toInt() }
+        val operators = elements[false]!!
```

Comment:
만약 `operators`가 `null`이라면 어떤 위험이 발생할까요? 그 위험을 피할 수 있는 방법도 찾아보면 좋을 것 같아요.

```suggestion
        val operators = elements.getOrDefault(false, emptyList())
```

Reply:
감사합니다! :) 
해당 Null 체크는 빈 배열이면 로직상 문제가 있어서 Early Exit으로 처리하였습니다 :)